Sources/ArcGISToolkit/Components/OfflineMapAreasView/OfflineMapAreasView.swift

```diff
@@ -56,12 +62,20 @@ public struct OfflineMapAreasView: View {
             }
             .task {
                 await mapViewModel.makePreplannedOfflineMapModels()
+                mapViewModel.loadPreplannedMobileMapPackages()
             }
             .toolbar {
                 ToolbarItem(placement: .confirmationAction) {
                     Button("Done") { dismiss() }
                 }
             }
+            .onAppear {
+                UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .sound]) { _, error in
```

Comment:
Did you choose the escaping closure version of this method over the try-await one because of Mac Catalyst compatibility? I noticed that the example project doesn't specify Mac Catalyst as one of the destinations, so want to double check.

Reply:
I used this closure based on the same implementation in the JobManagerExample. I did not add Mac Catalyst support to the offline example project, I'm not sure we need it. The component runs on Mac Catalyst similar to other Toolkit components.